oculus_ros2/package.xml

```diff
@@ -13,21 +13,35 @@
   <build_depend>roscpp</build_depend>
   <build_depend>rospy</build_depend>
   <build_depend>std_msgs</build_depend>
+  <build_depend>sensor_msgs</build_depend>
```

Comment:
All the build + build_export + exec dependencies can be listed as <depend>

Reply:
I think your sentence is not finished, in any case I do not understand what I can do.

Reply:
All the build + build_export + exec dependencies can be listed as `<depend>`